ext/yarp/extconf.rb

```diff
@@ -56,8 +56,16 @@ def build_static_rubyparser
       end
 
       def build_target_rubyparser(target)
+        # explicit "sh" is used for Windows where shebangs are not supported
         Dir.chdir(root_dir) do
-          Rake.sh("sh configure") # explicit "sh" for Windows where shebangs are not supported
+          if !File.exist?("configure") && Dir.exist?(".git")
+            # this block only exists to support building the gem from a "git" source,
+            # normally we package up the configure and other files in the gem itself
+            Rake.sh("sh autoconf")
+            Rake.sh("sh autoheader")
+            Rake.sh("rake templates")
+          end
+          Rake.sh("sh configure")
```

Comment:
```suggestion
            Rake.sh("sh", "autoconf")
            Rake.sh("sh", "autoheader")
            Rake.sh("rake","templates")
          end
          Rake.sh("sh", "configure")
```

We can avoid a subshell here. Not the end of the world but it's one of my pet peeves.